controller/ca/controller.go

```diff
@@ -154,56 +146,29 @@ func (c *CertificateController) syncSecret(key string) error {
 			pkgK8s.TLSPrivateKeyFileName: certAndPrivateKey.PrivateKey,
 		},
 	}
-	secrets := c.k8sAPI.Client.CoreV1().Secrets(identity.Namespace)
-	_, err = secrets.Create(secret)
+	_, err = c.k8sAPI.Client.CoreV1().Secrets(identity.Namespace).Create(secret)
 	if apierrors.IsAlreadyExists(err) {
-		_, err = secrets.Update(secret)
+		_, err = c.k8sAPI.Client.CoreV1().Secrets(identity.Namespace).Update(secret)
```

Comment:
Is this just a stylistic change or is there a semantic change here?

Reply:
Just stylistic -- I wanted it to be consistent with how we're handling the configmap write on lines 114-117 of this file.